src/samples/audio/detect_test.c

```diff
@@ -199,6 +199,7 @@ static void default_detect_test(struct comp_dev *dev,
 		}
 	}
 }
+
 static int test_keyword_get_threshold(struct comp_dev *dev, int sample_width)
```

Comment:
Unneeded new line.

Reply:
It was there originally and in one previous commit I've deleted it. For this reason I readded it here. What should I do now? To delete it?

Reply:
Either delete it or have a commit that deals with coding style which has this change. But if it's only this it's probably not really worth it to have the extra commit.